src/components/History/History.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import {StepButton} from "../Buttons/StepButton";
+import styles from './History.module.css';
+
+export function History({jumpTo}) {
+    const jumpBack = () => {
+        jumpTo(false)
+    }
+    const jumpNext = () => {
+        jumpTo(true)
+    }
```

Comment:
Rework it with one function with boolean parameter

Reply:
Fixed